### PR TITLE
LPS-141370

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LPS-142031

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 1]
```diff
@@ -199,12 +199,40 @@ definition {
...
+		}
+
+		task ("Go to Display Page Templates") {
+			ProductMenu.gotoPortlet(
+				category = "Design",
+				portlet = "Page Templates");
+
+			NavItem.gotoPageTemplates();
+
+			Staging.gotoNavTab(tabName = "Display Page Templates");
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:215]: why staging?

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 2]
```diff
@@ -379,21 +407,53 @@ definition {
...
 
-		// TODO Create the automation about the LPS-141370.
+		task ("Go to Display Page Templates") {
+			ProductMenu.gotoPortlet(
+				category = "Design",
+				portlet = "Page Templates");
+
+			NavItem.gotoPageTemplates();
+
+			Staging.gotoNavTab(tabName = "Display Page Templates");
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:423]: why Staging?

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 3]
```diff
@@ -379,21 +407,53 @@ definition {
...
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ViewBlogListinBlogDetailView {
+		task ("Go to Masterclass Site") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-141370.
+		task ("Go to Blog and Assert the List of Blogs Presence") {
+			Navigator._gotoPage(pageName = "Blog");
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:448]: Macros with _ should only be used inside the files where they are defined. Try to find another similar method in Navigator

**Review by david-gutierrez-mesa — CHANGES_REQUESTED** [review]

**Comment by david-gutierrez-mesa** [comment]
ci:forward

**Pull request closed without merging**